templates/library.html

```diff
@@ -9,17 +9,17 @@
     <h1 style="margin-top: 00px;" id="libraryName">{{library.name}}</h1>
     {{#watch}}
     <ul class="list-inline">
-      <li><i class="fa fa-eye"></i><strong> {{watch}}</strong></li>
-      <li><i class="fa fa-star"></i><strong> {{star}}</strong></li>
-      <li><i class="fa fa-code-fork"></i><strong> {{fork}}</strong></li>
+      <li><i class="far fa-eye"></i><strong> {{watch}}</strong></li>
+      <li><i class="fas fa-star"></i><strong> {{star}}</strong></li>
```

Comment:
I suppose we should use same style on the same block?

Reply:
| Solid | Regular |
|------|--------- |
| ![image](https://user-images.githubusercontent.com/12371363/61185092-fc2c5580-a64c-11e9-897a-0387ea9dd898.png) | ![image](https://user-images.githubusercontent.com/12371363/61185093-ffbfdc80-a64c-11e9-80cf-5732623d9c90.png) |
| ![image](https://user-images.githubusercontent.com/12371363/61185095-08b0ae00-a64d-11e9-9025-4c98f52cd878.png) | ![image](https://user-images.githubusercontent.com/12371363/61185097-0bab9e80-a64d-11e9-87e5-6ceaed9d57e6.png) |

I personally find the solid star to work better and the regular eye - the solid eye feels too bloated?

Reply:
In this case, maybe align to solid style IMO, should make the project more maintainable, thanks.